config/initializers/content_security_policy.rb

```diff
@@ -5,12 +5,22 @@
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
 Rails.application.config.content_security_policy do |policy|
+  # Attributes clarification
+  # :self, only allow resources from current origin
+  # :none, won't allow loading of any resources
+  #
+  # :https, requires secure http to load resources
```

Comment:
Kommentar korrigieren, etwa so:
```suggestion
  # :https, allow loading resources from arbitrary hosts that are protected by https
```